exercises/K-functions-parameters/exercise.js

```diff
@@ -1,9 +1,11 @@
 // Complete the function so that it takes input parameters
-function multiply() {
+function multiply(a,b) {
```

Comment:
It's customary to leave a space between parameters in the parameter list as `(a, b)`

Reply:
I fixed it. Thank you!